bodhi/tests/server/views/test_generic.py

```diff
@@ -53,6 +53,16 @@ def test_home(self):
         self.assertIn('Log out', res)
         self.assertIn('Fedora Update System', res)
 
+    @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
+    def test_critical_update_link_home(self):
+        update = Update.query.first()
+        update.critpath = True
+        update.status = UpdateStatus.testing
+        self.db.commit()
+        
+        res = self.app.get('/',headers={'Accept': 'text/html'})
+        self.assertRegexpMatches(str(res),'status=testing&critpath=True.*Latest Critical Path Updates in Need of Testing')
```

Comment:
```flake8``` wants a space after the , on this line and also wants the line to be less than 101 characters:

```
pip	./bodhi/tests/server/views/test_generic.py:64:42: E231 missing whitespace after ','
pip	        self.assertRegexpMatches(str(res),'status=testing&critpath=True.*Latest Critical Path Updates in Need of Testing')
pip	                                         ^
pip	./bodhi/tests/server/views/test_generic.py:64:101: E501 line too long (122 > 100 characters)
pip	        self.assertRegexpMatches(str(res),'status=testing&critpath=True.*Latest Critical Path Updates in Need of Testing')
pip	          
```